indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -143,3 +145,16 @@ class RHRoomSuggestions(RHRoomBookingBase):
     @use_args(search_room_args)
     def _process(self, args):
         return jsonify(get_suggestions(args, limit=NUM_SUGGESTIONS))
+
+
+class RHBookingDetails(RHRoomBookingBase):
+    def _process_args(self):
+        self.booking = Reservation.get_one(request.view_args['booking_id'])
+
+    def _process(self):
+        attributes = reservation_details_schema.dump(self.booking).data
+        date_range, occurrences = get_booking_occurrences(self.booking)
+        date_range = [dt.isoformat() for dt in date_range]
+        serialized = {dt.isoformat(): reservation_details_occurrences_schema.dump(data).data
```

Comment:
i am a little bit fussy here but you could easily bind to `occurrences` as well ;) `serialized` is not very explanatory